### PR TITLE
Bash scripts to initialize the project

### PR DESCRIPTION
**Pull request by Screenfeed**
## Scripts to install the WordPress test suite
A php script has been added, allowing to fix a compatibility issue with the tests suite: it does a text replacement in a file to make a method compatiblie with the one it extends.

## Scripts to install the required plugins and themes
Functions have been added to easy plugins and themes installations.

## Scripts for building and distributing the project
Basically the same things already present in Polylang, but with some parameters.

### CODE REVIEW

**Comment by Screenfeed** [comment]
@Chouby We probably want to do something about the composer scripts `install-suite` and `install-suite-nodb`, as it may not work for everybody :(
Any thoughts?

**Review by Chouby — CHANGES_REQUESTED** [review]

File: bin/build.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	COMPOSER_COMMAND='update'
+fi
+
+echo "Installing PHP packages..."
+composer $COMPOSER_COMMAND # Need update to ensure to have the latest version of the dependencies.
+
+if [[ $2 ]]; then
+	echo "Running build..."
+	npm install && npm run build # minify js and css files.
+fi
```
[the remark this answers is not in the document]
Comment (Screenfeed) [reply bin/build.sh]: Done in a739479.

File: bin/distribute.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+rm -rf vendor/ # Make sure to remove all traces of development dependencies.
+
+echo "Installing PHP packages..."
+composer update --no-dev # composer.lock file is always present because pushed on repository.
+
+if [[ $2 ]]; then
+	echo "Running build..."
+	npm install && npm run build # minify js and css files.
+fi
```
[the remark this answers is not in the document]
Comment (Screenfeed) [reply bin/distribute.sh]: Done in a739479.

File: bin/distribute.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+rsync -rc --exclude-from=.distignore . "$1/" --delete --delete-excluded
+zip -r "$1.zip" "$1/*"
+rm -rf "$1/"
```
[the remark this answers is not in the document]
Comment (Screenfeed) [reply bin/distribute.sh]: Yes it's possible. For `polylang/polylang-pro` it will be `polylang-pro.zip` for example. The caveat is for projects like this one: it will be `wp-phpunit.zip`, so a parameter can be used to customize this into `polylang-phpunit.zip` for example.
Done in a739479.

File: bin/install-wp-tests.sh [hunk 4]
```diff
@@ -150,6 +153,6 @@ install_db() {
 	mysqladmin create $DB_NAME --user="$DB_USER" --password="$DB_PASS"$EXTRA
 }
 
-install_wp
-install_test_suite
-install_db
+installWp
+installTestSuite
+installDb
```
[the remark this answers is not in the document]
Comment (Screenfeed) [reply bin/install-wp-tests.sh]: Done in a739479.

**Review by Chouby — CHANGES_REQUESTED** [review]
For the file `install-wp-tests.sh`, I propose to use directly the latest version of the original file available https://github.com/wp-cli/scaffold-command/blob/master/templates/install-wp-tests.sh

**Comment by Screenfeed** [comment]
Done in 1f113e8.
